homeassistant/components/braviatv/config_flow.py

```diff
@@ -102,15 +108,12 @@ async def async_step_authorize(
 
         if user_input is not None:
             try:
-                await self.init_device(user_input[CONF_PIN])
+                self.device_config[CONF_PIN] = user_input[CONF_PIN]
```

Comment:
Please only wrap the line that can raise in the try... except block. We can move this line up.